ipatests/test_webui/ui_driver.py

```diff
@@ -1246,7 +1260,8 @@ def find_record(self, entity, data, facet='search', dummy='XXXXXXX'):
 
     def add_record(self, entity, data, facet='search', facet_btn='add',
                    dialog_btn='add', delete=False, pre_delete=True,
-                   dialog_name='add', navigate=True, combobox_input=None):
+                   dialog_name='add', navigate=True, combobox_input=None,
+                   assert_on=True):
```

Comment:
As we discussed offline, let's use `negative=False` instead of `assert_on=True` to be consistent with other methods which use `negative` for expected failure.

Reply:
fixed